examples/00_quick_start/sasrec-tf2.ipynb

```diff
@@ -0,0 +1,321 @@
+{
```

Comment:
<p>a minor detail, in other notebooks we have a final markdown section named References, where we put the details of the paper and maybe other references that you might consider:</p><p>https://github.com/microsoft/recommenders/blob/main/examples/00_quick_start/lstur_MIND.ipynb</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1530/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>Added a reference section. </p>